### PR TITLE
Review development

### PR DESCRIPTION
**Pull request by squimrel**

### CODE REVIEW

File: include/iso9660.h [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+   */
+  std::string identifier;
+  int version;
+};
+
+/**
+ * The "generic" volume descriptor for the primary and supplementary volume
+ * descriptor.
+ */
+struct VolumeDescriptor {
```
Comment (MartinBriza) [inline include/iso9660.h]: I don't particularly like the overall structure of this file. I guess a bit more of object-oriented design would help with the readability a lot.
For example this structure should (at least in my opinion) be a class and the function that creates it should be its constructor. More complicated operations on it then can be implemented as its methods.
Same goes for the structs above, too.

**Comment by MartinBriza** [comment]
Overall I'd say LGTM, one thing I'd do is move the methods around a bit so it's more object oriented.

File: include/iso9660.h (hunk 1 above)
Comment (squimrel) [reply include/iso9660.h]: +1

**Comment by squimrel** [comment]
Once this is merged it'll be tagged as version 0.0.1 so that I can try out the spec file build properly.

**Pull request merged**